daemon/test-startup.c

```diff
@@ -263,6 +284,8 @@ main (int argc, char **argv)
 
 	g_test_add ("/daemon/startup/replace", Test, NULL,
 	            setup, test_daemon_replace, teardown);
+	g_test_add ("/daemon/startup/ssh-agent", Test, NULL,
```

Comment:
that doesn't look to me like an ECDSA-specific test case, wouldn't it be better to introduce it before ECDSA changes? (so it's easier to verify that it worked before and after the changes were introduced)